free-1.md

```diff
@@ -4,40 +4,130 @@
+would consider any kind of computer aided control tool to be a form of cheating
+since it provides users with an unfair advantage. This advantage is made very
+clear in
+[this comparison video](https://www.youtube.com/watch?v=DXUOWXidcY0)
+between a top tier professional player and a bot designed to do the same task.
+
+Furthermore, although compititions exist to design AIs to play the games,
+one such is UC Berkeley's "Berkeley Overmind" (as seen in 
+[this video](https://www.youtube.com/watch?v=fWtpTu9gB20)) which plays
+Starcraft with a superhuman average actions per minute (APM) of 3397 and a
+peak performance over 7000 APM, no game to my knowledge incorperates coding
+natively for fine grained unit control.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Since this language aims to allow gamers to control units beyond the bounds of
+a typical human reaction time, there must be some coding interface to allow
+users to express their desired behaviors in game. Thus, by allowing users
+to express a controls in code form, they can be sped up to superhuman speeds.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+Although this concept makes for an interesting game idea, the real reason I
+am excited about designing this game is because of its educational value. I
+started playing Starcraft back in 4th grade and its modding tools 
+were my original introduction to programming. Although this game should be
+ideally enjoyable to all programmers, I imagine if the levels are designed
+well with a good progression in difficult, this game could be used to introduce
+completely novice programmers to coding. I've always been amazed with how
+even the most monotonous tasks can be made "fun" in the context of a video
+game by occasionally adding some flashing lights and a "level-up" screen.
+Although I personally think programming is fun to begin with, I think by
+wrapping core coding skills within a enteraining game will help coding
+become more accessable to gamers.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Hardcore gamers teach novice programming.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+As mentioned above, I planned to implement a number of "languages" in the game.
+Originally, I thought making a language that resembled C, Python,
+or JavaScript would be the most "useful" to students since a majority of
+introductory programming classes start with these languages. That being said,
+I have also seen languages like
+[Scratch](https://scratch.mit.edu/) and
+[Blockly](https://developers.google.com/blockly/) which are more targeted
+towards novice coders. In reality, any of these will work and I will probably
+implement some varient of them all.
```

Comment:
Would it possibly be easier to do Scala if you are writing in Scala? It's true that it's not your typical "intro programming language" but for ease of coding, is this a good thing to consider?